controllers/playlists.controller.js

```diff
@@ -0,0 +1,109 @@
+const { Playlist } = require('../models/playlist.model');
+const { extend } = require('lodash');
+
+const createNewPlaylist = async (req, res) => {
+	try {
+		let newPlaylist = req.body;
+		newPlaylist = new Playlist(newPlaylist);
```

Comment:
I think using different variable names would be better in readability here. like requestBody, newPlaylist, populatedPlaylist.